terraform/main.tf

```diff
@@ -102,3 +102,297 @@ resource "google_storage_bucket_iam_binding" "binding" {
     "user:aengel@rmi.org",
   ]
 }
+
```

Comment:
I'd like to learn more about how to structure terraform projects.  It'd be helpful to isolate the superset setup in a separate file or something so we can easily reuse it for other projects like the pudl usage metrics repo.